main.go

```
@@ -17,78 +19,81 @@ import (
+	TrustProxy      bool     `json:"trust_proxy,omitempty"`      // 是否信任代理头
+	SourceHeaders   []string `json:"source_headers,omitempty"`   // 自定义来源标头列表
+
+	blockedIPNets []*net.IPNet // 存储阻止的 IP 网段
+	mu            sync.RWMutex // 读写锁，用于保护阻止列表的并发访问
+	stopChan      chan struct{} // 用于停止阻止列表刷新协程
+	logger        *zap.Logger   // 日志记录器
 }
 
-// CaddyModule 返回模块信息。
+// CaddyModule 返回模块的信息
 func (IPBlocker) CaddyModule() caddy.ModuleInfo {
```

golangci-lint (1.62.2)

copylocks: CaddyModule passes lock by value: github.com/WJQSERVER/caddy-ip-blocker.IPBlocker contains sync.RWMutex

(govet)